src/tbp/interactive/tutorials/simple_visualization/tutorial.md

```diff
@@ -0,0 +1,876 @@
+        )
+        self.plotter.at(2).show(
+            axes=deepcopy(self.axes_dict),
+            interactive=True,
+            resetcam=True,
+        )
+
+
+@register(
+    "interactive_tutorial",
+    description="Simple tutorial with four widgets",
+)
+def main(experiment_log_dir: str, objects_mesh_dir: str) -> int:
+    """Interactive tutorial visualization.
+
+    Args:
+        experiment_log_dir: Path to the experiment directory containing the detailed
+            stats file.
+        objects_mesh_dir: Path to the root directory of YCB object meshes.
+
+    Returns:
+        Exit code.
+    """
+    if not Path(experiment_log_dir).exists():
+        logger.error(f"Experiment path not found: {experiment_log_dir}")
+        return 1
+
+    data_path = str(Path(objects_mesh_dir).expanduser())
+
+    InteractivePlot(experiment_log_dir, data_path)
+
+    return 0
+
+
+@attach_args("interactive_tutorial")
+def add_arguments(p: argparse.ArgumentParser) -> None:
+    p.add_argument(
+        "experiment_log_dir",
+        help=(
+            "The directory containing the experiment log with the detailed stats file."
+        ),
+    )
+    p.add_argument(
+        "--objects_mesh_dir",
+        default="~/tbp/data/habitat/objects/ycb/meshes",
+        help=("The directory containing the mesh objects."),
+    )
+```
+
+### Registering the Plot with `@register`
+
+The `@register` decorator declares a new plot entry named `interactive_tutorial`.
+This name is the handle you will use from the plotting CLI to run this plot.
+The registry scans the files under `src/tbp/plot/plots/` and collects any functions decorated with `@register`.
+The registered `main` function is the plot’s entrypoint, where all the plotting code is placed.
+
+### Attaching CLI Arguments with `@attach_args`
+
+The `@attach_args` decorator links an argument definition function to the plot.
+This allows the user to add argparse arguments which will be passed to the function `main`.
+
+In this tutorial we define one required argument, `experiment_log_dir`,
+and one optional argument, `--objects_mesh_dir`, which defaults to the YCB mesh directory.
+
+### The `InteractivePlot` Class
+
+Inside `InteractivePlot.__init__`, we instantiate all the classes that the widgets will share.
+
+- `DataParser(exp_path)` loads and provides access to the experiment log.
+Later widgets will use locators to query and extract episode and step data from it.
+
+- `YCBMeshLoader(data_path)` provides the ability to load textured meshes for YCB object IDs.
+We initialize it here so any mesh visualization widget can reuse it.
+
+- `Publisher()` creates the shared event bus.
+In later sections, widgets will publish topic messages to this bus and subscribe to them through updaters.
+
+- `VtkDebounceScheduler(plotter, period_ms)` provides a central debounce mechanism so widget updates can be throttled.
+Interactive widgets can produce a high volume of callbacks, especially while dragging sliders.
```

Comment:
```suggestion
Interactive widgets can produce a high volume of callbacks, especially while dragging sliders; throttling with a debounce mechanism ensures the plot updates smoothly, rather than attempting to render computationally intense objects hundreds of times.
```

Suggestion for some wording to make this a bit more concrete to someone not familiar with this problem.